scripts/install-dependencies.sh

```diff
@@ -0,0 +1,81 @@
+#!/usr/bin/env bash
+
+GIT_REPO=jaxxzer/companion
+GIT_BRANCH=setup-clean
+GIT_TAG=0.1.0
+
+. ./bash-helpers.sh
+
+# Update package lists and current packages
+run_step export DEBIAN_FRONTEND=noninteractive
+APT_OPTIONS=-yq
+run_step sudo apt update $APT_OPTIONS
+run_step sudo apt upgrade $APT_OPTIONS
+
+# install python and pip
+run_step sudo apt install $APT_OPTIONS \
+  python-dev \
+  python-pip \
+  python-libxml2 \
+  python-lxml \
+  libcurl4-openssl-dev \
+  libxml2-dev \
+  libxslt1-dev \
+  libffi-dev \
+  git \
+  screen \
+  npm \
+  nodejs \
+  gstreamer1.0-tools \
```

Comment:
Maybe we should specify the versions like this:
`sudo apt-get install gstreamer1.0-tools=1.4.4-2`